### PR TITLE
@orta: initial basic styles via albers gem

### PR DESCRIPTION
**Pull request by dylanfareed**
We have a couple styleguide-like projects in use by teams internally. Though they are actively being worked on, neither are in a OS-ready state for the moment. 

In the interests of getting this app to a usable place, I've put a super minimal library together called [Albers](https://github.com/dylanfareed/albers)--based on [Bourbon](http://bourbon.io/) and [Neat](http://neat.bourbon.io/)--that works with Rails and Middleman apps. Albers provides bare minimum of coverage, but that's more than enough for the internal/trivial toolset that is Echo.

image

image

image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,6 +13,8 @@ gem 'braque'
 gem 'haml'
 gem 'simple_form'
 
+gem 'albers', git: 'https://github.com/dylanfareed/albers'
```
Comment (dylanfareed) [inline Gemfile:16]: Distribution directly via GitHub for now. Didn't really want to pollute the RubyGems namespace.

File: app/views/accounts/features/index.haml [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+      %td.actions
+        = link_to t('actions.edit'), edit_account_feature_path(@account, feature), class: 'btn btn-secondary btn-tiny'
+        = link_to t('actions.delete'), account_feature_path(@account, feature), method: :delete, data: { confirm: "Are you sure?" }, class: 'btn btn-delete btn-tiny'
+
+- if @features.try(:next_link) || @features.try(:previous_link)
+  .list-pager
+    - if @features.try(:previous_link)
+      = link_to 'Previous', account_features_path(@account, size: @size, page: @page - 1), class: 'list-pager-prev'
+    - if @features.try(:next_link)
+      = link_to 'Next', account_features_path(@account, size: @size, page: @page + 1), class: 'list-pager-next'
```
Comment (dylanfareed) [inline app/views/accounts/features/index.haml:24]: Markup for "components" like pagination could well be moved upstream to Albers down the road.

Comment (dylanfareed) [inline app/views/accounts/features/index.haml:24]: Also, yay, Braque/Gris make simple pagination pretty trivial. So let's add it here...

Comment (orta) [inline app/views/accounts/features/index.haml:24]: nice

**Comment by orta** [comment]
This `%` thing in HAML is interesting, I use [slim](http://slim-lang.com) for all of CocoaPods which doesn't have it.

**Comment by orta** [comment]
Looks good though, yo

**Pull request merged**